### ISSUE TITLE
Market on Planet OE-D2 sells ELECTROINICS.

### ISSUE THREAD

**Issue by bchoii** [SpaceTradersAPI/issues-and-suggestions#28]
Suspect typo in word ELECTROINICS .

**Comment by jadlers** [comment]
Adding the current response to [`api.spacetraders.io/game/locations/OE-D2/marketplace`](https://api.spacetraders.io/game/locations/OE-D2/marketplace)

```json
{
  "planet": {
    "symbol": "OE-D2",
    "type": "PLANET",
    "name": "Delta II",
    "x": 4,
    "y": 12,
    "marketplace": [
      {
        "symbol": "METALS",
        "pricePerUnit": 8,
        "available": 40613
      },
      {
        "symbol": "FUEL",
        "pricePerUnit": 13,
        "available": 19853
      },
      {
        "symbol": "ELECTROINICS",
        "pricePerUnit": 655,
        "available": 12265
      },
      {
        "symbol": "TEXTILES",
        "pricePerUnit": 28,
        "available": 6702
      },
      {
        "symbol": "CHEMICALS",
        "pricePerUnit": 11,
        "available": 191958189
      },
      {
        "symbol": "MACHINERY",
        "pricePerUnit": 97,
        "available": 675
      },
      {
        "symbol": "CONSTRUCTION_MATERIALS",
        "pricePerUnit": 354,
        "available": 98627
      },
      {
        "symbol": "WORKERS",
        "pricePerUnit": 17,
        "available": 8803
      },
      {
        "symbol": "FOOD",
        "pricePerUnit": 15,
        "available": 18597
      }
    ]
  }
}
```

**Comment by Stumblinbear** [comment]
Planet OE-D2 no longer exists, closing
